yt_dlp/extractor/audius.py

```diff
@@ -245,3 +245,32 @@ def _real_extract(self, url):
         return self.playlist_result(entries, playlist_id,
                                     playlist_data.get('playlist_name', title),
                                     playlist_data.get('description'))
+
+
+class AudiusProfileIE(AudiusPlaylistIE):
+    IE_NAME = 'audius:artist'
+    IE_DESC = 'Audius.co profile/artist pages'
+    _VALID_URL = r'https?://(?:www)?audius\.co/(?P<id>[^\/]+)/$'
```

Comment:
```suggestion
    _VALID_URL = r'https?://(?:www)?audius\.co/(?P<id>[^\/]+)/?(?:[?#]|$)'
```
These should also match
```
https://audius.co/pzl
https://audius.co/pzl?par=val
https://audius.co/pzl#something
```